appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd

```diff
@@ -0,0 +1,1163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
At https://jakarta.ee/xml/ns/jakartaee/#10 - it is `2009, 2021`.
So this isn't just copied and re-formatted.
What is the source of this file, then?

Reply:
As you commented later, the source of this file is [the file below https://github.com/eclipse-ee4j/jakartaee-schemas/tree/master/xml/src.](https://github.com/eclipse-ee4j/jakartaee-schemas/blob/master/xml/src/connector_2_1.xsds).